muckrock/agency/models/agency.py

```diff
@@ -244,15 +245,26 @@ def get_user(self):
         try:
             return self.profile.user
         except Profile.DoesNotExist:
-            data = {
+            # This is a temporary work around for not writing back to squarelet
+            uuid = uuid4()
+            user_json = {
+                "preferred_username": self.name[:150],
                 "name": self.name,
-                "preferred_username": self.name,
-                "is_agency": True,
+                "agency": self,
+                "email": "",
+                "uuid": uuid,
+                "organizations": [
+                    {
+                        "name": self.name,
+                        "slug": slugify(self.name),
+                        "entitlements": [],
+                        "max_users": 1,
+                        "individual": True,
+                        "uuid": uuid,
+                        "admin": True,
+                    }
+                ],
```

Comment:
I'm _guessing_ upstream Muckrock won't want this hacky fix, right?

Reply:
removed in https://github.com/MuckRock/muckrock/pull/1790/commits/bdc6523f026ac7b8d6e0fdfcff5c77cfb390a16c